Lib/test/test_ssl.py

```diff
@@ -4045,8 +4055,8 @@ def test_dh_params(self):
                                    chatty=True, connectionchatty=True,
                                    sni_name=hostname)
         cipher = stats["cipher"][0]
-        parts = cipher.split("-")
-        if "ADH" not in parts and "EDH" not in parts and "DHE" not in parts:
+        parts = set(cipher.split("-"))
+        if not dhe_aliases.intersection(parts):
```

Comment:
I think there is no need for `set(cipher.split("-"))` as `intersection` takes any iterable.

Reply:
Sure, I can remove that.

Reply:
Removed in 9bf7e6c.